runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -307,6 +334,27 @@ public void invalidateClientCache() {
         readCache.invalidateAll();
     }
 
+    /**
+     * Invalidate cache entries of which the keys are less than the specific address.
+     *
+     * @param address Keys less than the input log address will be invalidated.
+     */
+    public void invalidateClientCache(long address) {
+        // TODO: Might need to do some statistics to clear up cache when the amount to
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)